src/Apps/Artist/Routes/Overview/Components/ArtistCareerHighlights.tsx

```diff
@@ -21,63 +16,45 @@ interface ArtistCareerHighlightsProps {
 const ArtistCareerHighlights: React.FC<ArtistCareerHighlightsProps> = ({
   artist,
 }) => {
-  const { t } = useTranslation()
+  // const { t } = useTranslation()
 
-  const { credit, partner, text } = artist.biographyBlurb!
+  const hasCareerHighlights = artist.insights.length > 0
```

Comment:
So we'll want to take the remainder of the list that isn't being used in the header. So  `const insights = artist.insights.slice(6)` or similar, then use that throughout. We'll want to fetch the full list of insights as well, instead of a selection.

Reply:
Gotcha, will tackle that next!

Reply:
I just pushed a commit that does the slicing, but they won't make sense until there's a proper sort on the MP side of things.